components/automate-ui/src/app/entities/userperms/userperms.effects.ts

```diff
@@ -78,9 +78,12 @@ export class UserPermEffects {
   @Effect()
   fetchParameterizedPerms$ = this.actions$.pipe(
     ofType(UserPermsTypes.GET_PARAMETERIZED),
-    mergeMap((action: GetUserParamPerms) => this.requests.fetchParameterized(action.payload).pipe(
+    mergeMap((action: GetUserParamPerms) => {
```

Comment:
This change fixed the Custom test failure in the public buildkite test. 
Below is the error message
```
/go/src/github.com/chef/automate/components/automate-ui/src/app/entities/userperms/userperms.effects.ts
severity:error rule:go.src.github.com.chef.automate.semgrep.observables-from-select-or-pipe-not-ending-with-dollar-sign: Observable variable should end with a dollar sign.
81:action: GetUserParamPerms) => this.requests.fetchParameterized(action.payload).pipe
ran 11 rules on 725 files: 1 findings
```
Below is a link to the test failure


https://buildkite.com/chef-oss/chef-automate-master-verify/builds/14304#6e5fd421-00fb-4c92-8ba8-da8d9a27faab/93